task10.js

```diff
@@ -0,0 +1,17 @@
+// Write a function that calculates the average of elements in an array.
+
```

Comment:
That function correctly calculates the average of elements in an array.
Howewer exist few cases))
When array is empty your function will return `NaN`, but it is not correct
When some value is not a number then calculation will be wrong.